bgpd/bgp_mplsvpn.c

```diff
@@ -963,17 +963,17 @@ DEFUN (show_bgp_ip_vpn_rd,
        "VPN Route Distinguisher\n"
        JSON_STR)
 {
-  int idx_ext_community = 5;
+  int idx_rd = 5;
```

Comment:
idx_rd is not going to be 5.  

If I enter:

'show ip bgp ipv4 vpn all rd ASN:nn_or_IP-address:nn"
shouldn't we be at 6?